polaris-react/src/components/KeyboardKey/KeyboardKey.scss

```diff
@@ -1,23 +1,32 @@
-$key-base-height: 24px;
+$key-base-dimension: 28px;
 
 .KeyboardKey {
   // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  display: inline-block;
-  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  height: $key-base-height;
+  height: $key-base-dimension;
+  display: inline-flex;
+  justify-content: center;
   margin: 0 var(--p-space-05) var(--p-space-05);
-  padding: 0 var(--p-space-1);
-  background-color: var(--p-surface);
-  box-shadow: 0 0 0 var(--p-space-025) var(--p-border-subdued),
-    0 var(--p-space-05) 0 0 var(--p-surface),
-    0 var(--p-space-05) 0 var(--p-space-025) var(--p-border-subdued);
-  border-radius: var(--p-border-radius-1);
-  font-family: var(--p-font-family-sans);
-  font-size: var(--p-font-size-75);
+  padding: 0 var(--p-space-2);
+  background: var(--p-surface-depressed);
+  box-shadow: 0 var(--p-space-025) 0 var(--p-hint-from-direct-light);
+  border-radius: var(--p-space-1);
```

Comment:
This should use a [border-radius token](https://github.com/Shopify/polaris/blob/main/polaris-tokens/src/token-groups/shape.ts#L34)